src/3d/qgscameracontroller.h

```diff
@@ -189,6 +189,21 @@ class _3D_EXPORT QgsCameraController : public QObject
      */
     void rotateCamera( float diffPitch, float diffYaw );
 
+    /**
+     * Rotates the camera around the pivot point (in world coordinates)
+     * to the given new pitch and heading angle.
+     * \since QGIS 3.42
+     */
+    void rotateCameraAroundPivot( float newPitch, float newHeading, const QVector3D &pivotPoint );
+
+    /**
+     * Zooms camera by given zoom factor (higher than one means zoom in)
```

Comment:
I wouldn't normally be this pedantic, but since the tests are failing anyway and it'll need another run through CI regardless: :stuck_out_tongue_closed_eyes: 

I initially misinterpreted this as meaning something about the "camera being higher than ... "

```suggestion
     * Zooms camera by given zoom factor (>1 means zoom in)
```

Reply:
fixed in [783e962](https://github.com/qgis/QGIS/pull/59904/commits/783e962e3b54d8a465f0fd5ac1a6243a50ebe613)